.rubocop.yml

```diff
@@ -0,0 +1,39 @@
+# use the shared YaST defaults
+inherit_from:
+  /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
+
+# Redundant returns add legibility for developers used to other languages
+Style/RedundantReturn:
+  Enabled: false
+
+# Don't enforce any particular name for block params
+SingleLineBlockParams:
+  Enabled: false
+
+# The general YaST agreement is 100 characters, so it fits into standard
+# resolutions and Github's pull request view. But let's open the threshold a
+# little bit
+Metrics/LineLength:
+  Max: 105
+
+# Enforce if/unless at the end only for really short lines
+Style/IfUnlessModifier:
+  MaxLineLength: 60
+
+# some storage API have size method, but without empty? method
+# for details see https://github.com/yast/yast-storage-ng/pull/83
+Style/ZeroLengthPredicate:
+  Enabled: false
+
+# the ".freeze" attribute for the constants is not nice
```

Comment:
hmm, we have this discussion also in storage and in fact, freeze is what make constant real constant